app/assets/stylesheets/style.scss

```diff
@@ -14,6 +14,13 @@ body {
 label {
   color: white;
 }
+a {
+  color: black;
+    background:rgba(31,138,150,0.95);
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Commas in function arguments should be followed by a single space<br>Colon after property should be followed by one space